dask_cloudprovider/aws/ecs.py

```diff
@@ -953,6 +967,18 @@ async def _start(
                 or await self._create_security_groups()
             )
 
+        self._worker_task_def_name = (
+            f"{str(self._task_def_name)}-worker-{uuid}"
+            if self._task_def_name
+            else "dask-worker-{uuid}"
+        ).format(uuid=str(uuid.uuid4())[:5])
```

Comment:
In other parts of Dask we also unpack `os.environ` into formats like these which allows users to incorporate environment variables into their config. For example they could set the task def name to `"dask-worker-task-{USER}-{uuid}"` to include their username.

```suggestion
        ).format(uuid=str(uuid.uuid4())[:5], **os.environ)
```